src/main/java/ua/org/crazy/homework02/BigFactorial.java

```diff
@@ -0,0 +1,18 @@
+package ua.org.crazy.homework02;
+
+import java.math.BigInteger;
+
+public class BigFactorial {
+
+    public static void main(String[] args) {
```

Comment:
It is better to separate logic and method main. Put logic into a separate class.